client/.env.local

```diff
@@ -0,0 +1 @@
+REACT_APP_HOST_API=http://localhost:4000/v1/
```

Comment:
Check the [precedence rules](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#what-other-env-files-can-be-used), this will always overwrite the other one.

Reply:
🤦🏽‍♀️ i over confidently read that the wrong way round